### DISCUSSION TITLE
修課結束後看課程教材

### DISCUSSION THREAD

**Discussion by stan890425** [1101-GS4719-C/Python-Community-GS4719-C#654] [Show and tell]
如果修完課之後還想找上課教材，
可以到eeclass找到之前課程的教材。
https://tw.formosasoft.com/faq/819

**Comment by leon10456** [comment]
感謝樓主的分享 終於不會在學期結束好教材通通不見

**Comment by 110409538** [comment]
之前以為課程結束後就找不到資料了,感謝分享!

**Comment by HsuChen88** [comment]
這門課老師補充了非常多的教材
而且都是整理過的 *超棒*
還好之後不會不見
以後每次需要的時候就可以再上來找了

**Comment by gnsJhenJie** [comment]
不會不見的前提是學校不會過一陣子就換個系統(中央的學習系統已經從LMS變到eeclass再變到現在的新eeclass了), 感覺沒隔幾年就換一個系統, 也不是用原網站升級的方式...

**Comment by Yao900108** [comment]
感謝分享，之前也有在煩惱這個問題，沒想到還可以這樣!

**Comment by Jimmy911218** [comment]
謝謝樓主分享~我以後就不用一直下在東西了

**Comment by evanchou1204** [comment]
現在才發現有這個功能，可以複習以前的教材真的很棒！

**Comment by Yunhans** [comment]
感謝分享
我還真的不知道
可以回去看之前ai和java學了什麼我都忘光光了

**Comment by Pietra1226** [comment]
感謝分享，可以好好來複習了

**Comment by dabbie109303501** [comment]
我也是這學期才從室友那邊知道的
有種大學前2年都白念的感覺 😂
